recipes/ponio/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ponio" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hpc-maths/ponio/archive/v{{ version }}.tar.gz
+  sha256: 758929cf49509896063a1a3e989423debb60ce149e892c4c9130f8f7f6dc5ac4
+
+build:
```

Comment:
```suggestion
build:
  noarch: generic
```

Reply:
I don't understand the addition of the line `noarch: generic`, the bot `conda-forge-webservices` tells me thath it must be removed...

This library is header only but I have to generate a file in Python during installation (hence the dependencies to jinja2 for example).

Reply:
The bot is wrong here. It probably runs into an edge case during linting. 

`noarch: generic` says that this package is not architecture specific, i.e. a single build of it can be installed on any platform.